community/server/src/main/java/org/neo4j/server/configuration/ServerSettings.java

```diff
@@ -180,6 +180,14 @@ private ThirdPartyJaxRsPackage createThirdPartyJaxRsPackage( String packageAndMo
     public static final Setting<Duration> transaction_idle_timeout = setting( "dbms.rest.transaction.idle_timeout",
             DURATION, "60s" );
 
+    @Description( "Publicly discoverable bolt:// URI to use for Neo4j Drivers wanting to access the data in this " +
+            "particular database instance. Normally this is the same as the advertised address configured for the " +
+            "connector, but this allows manually overriding that default." )
+    @DocumentedDefaultValue(
+            "Defaults to a bolt://-schemed version of the advertised address " + "of the first found bolt connector." )
+    public static final Setting<URI> bolt_discoverable_address =
```

Comment:
Can this setting be made internal + unsupported for now?
Why does this setting live in `ServerSettings`? Shouldn't it be together with other connector settings?